doc/requirements.md

```diff
@@ -0,0 +1,34 @@
+# Requirements
+
+## Owner 
+ - don't want to store big images online
+ - moderation
+   - want to be able to delete QR codes that are bad or malicious 
+   - able to delete players 
+
+## Player
+attributes? (e.g. Username and contact information (like what contact info tho?)
+ - Do not want to have to login to personal application using username/password as device ID (SIM, phone, w/e) can identify me 
+   - BUT use log in to add additional devices(?)
+ - QR code is scanned by the camera (camera access button) -- don't record the actual code
+ - Can decline recording geolocation for privacy reasons
+ - Add new QR codes
+ - When scanning QR codes -- record a photo of the location or object + geolocation of the location or object
+ - Able to add comments to the QR codes 
+ - Able to see that other players have scanned the same QR code 
+ - QR codes inventory management (browse QR codes currently owned)
+    - Remove QR codes collected
+    - Sorting algorithm to sort the QR code list from higher to lowest or lowest to highest
+      - we can use this everywhere there's a QR code list
+ - Show sum of the QR code values that have been scanned
+ - Total number of QR codes that have been scanned
+ - See other players profiles (search by usernames)
+ - Browse QR codes that other players have scanned
+ - Create/Generate QR codes -- that show player's game status (the person who had made the QR code) -- saves in their library so that they can post it later?
+ - Create/Generate QR code to log in to another device with the same account - 
+ - See a map of geo-locations of nearby QR codes
+    - Search for nearby QR codes by using geo location
+ - Ability to see game-wide high scores of all players
+ - Estimate player's ranking for highest scoring unique QR code scanned (may be displayed in the user profile)
+ - Estimate player's ranking for total number of QR codes scanned (in user profile?)
+ - Estimate player's total sum of scores of QR codes scanned
```

Comment:
should this not be calculate these seem easily quantifiable values

Reply:
I assume you're mentioning this because I saw "Estimate"--I mostly say estimate because that's what the project description said, but yeah this should be pretty easily quantified.